parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -564,7 +564,13 @@ configGPUDrivers() {
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
         retrycmd_if_failure 120 5 25 nvidia-modprobe -u -c0 || exit $ERR_GPU_DRIVERS_START_FAIL
     fi
-    retrycmd_if_failure 120 5 25 nvidia-smi || exit $ERR_GPU_DRIVERS_START_FAIL
+    
+    # Noticed that NVIDIA driver version R515 and later versions will experience long loading time of nvidia-smi on 
+    # ND A100-v4 and NDm A100-v4 series. For now, use the workaround solution to enable NVIDIA persistence mode which 
+    # will accelerate the loading time of future nvidia-smi operations
+    retrycmd_if_failure 120 5 35 nvidia-persistenced --persistence-mode || exit $ERR_GPU_DRIVERS_START_FAIL
```

Comment:
https://github.com/Azure/AgentBaker/pull/2829/files#r1143546871

not sure this is sufficient for user scenarios, it just makes provisioning work